package.json

```diff
@@ -61,7 +61,9 @@
     "semver": "~5.5.1",
     "swagger-parser": "~6.0.5",
     "ws": "^6.2.2",
-    "yauzl": "~2.10.0"
+    "yauzl": "~2.10.0",
+    "yawn-yaml": "~1.5.0",
+    "lodash": "~4.17.21"
```

Comment:
Less libraries is always better, and reputable libraries are best. We already use lodash in other parts of the code so I guess thats ok, but why use yawn-yaml instead of the standard yaml library?